app/models/commits.rb

```diff
@@ -0,0 +1,17 @@
+class Commits
+
+  attr_reader :author, :message, :date
+
+  def initialize(attributes={})
+    @author = attributes['commit']['author']['name']
+    @message = attributes['commit']['message']
+    @date = attributes['commit']['author']['date']
+  end
+
+  def self.find_commits(username, repository)
```

Comment:
maybe `find_all` is a better name here? in our controller then, we can do `Commit.find_all()`